### PR TITLE
NS1. API support

### PR DESCRIPTION
**Pull request by justmwa**
added https://ns1.com API support

### CODE REVIEW

**Comment by Neilpang** [comment]
Please fix the format:  https://travis-ci.org/Neilpang/acme.sh/jobs/209126152

File: README.md [hunk 1]
```diff
@@ -308,6 +308,7 @@ You don't have to do anything manually!
 1. Domain-Offensive/Resellerinterface/Domainrobot API
 1. Gandi LiveDNS API
 1. Knot DNS API
+1. NS1. API
```
Comment (Neilpang) [inline README.md]: NS1.com API

**Comment by Neilpang** [comment]
@justmwa 
How is going on ?

**Comment by Neilpang** [comment]
@justmwa 
Any updates ?

**Comment by justmwa** [comment]
OK now ? Got issues with git merge then forgot about this 


De: "neil" <notifications@github.com> 
À: "Neilpang/acme.sh" <acme.sh@noreply.github.com> 
Cc: "justmwa" <dev@1e.ca>, "Mention" <mention@noreply.github.com> 
Envoyé: Lundi 8 Mai 2017 15:18:23 
Objet: Re: [Neilpang/acme.sh] NS1. API support (#714) 



[ https://github.com/justmwa | @justmwa ] 
Any updates ?

**Comment by Neilpang** [comment]
Thanks.

File: dnsapi/README.md [hunk 2]
```diff
@@ -394,6 +394,18 @@ acme.sh --issue --dns dns_knot -d example.com -d www.example.com
 
 The `KNOT_SERVER` and `KNOT_KEY` settings will be saved in `~/.acme.sh/account.conf` and will be reused when needed.
 
+
+## 20. Use NS1.com API
```
Comment (Neilpang) [inline dnsapi/README.md]: your number is not `20` now.
Please move to the last.

**Comment by Neilpang** [comment]
It seems there are some merge errors in your code.
please check.

**Pull request merged**